docs/key-registry.md

```diff
@@ -0,0 +1,93 @@
+# Key Registry
+
+## Sequence Diagram
+
+TODO: add sequence diagram
+
+## Basics
+
+### What is the key registry?
+
+A key registry is a list of keys stored by a client. A key is generated and added to the registry by a client and exposed publically at the path `/jwks.json` in anticipation of a grant request on an authorization server (AS).
```

Comment:
Maybe elaborate on what a client is? I think we should at least mention that a client is represented by a payment pointer and that the keys are at the path `PAYMENT_POINTER/jwks.json`